control-spec.txt

```diff
@@ -1143,6 +1143,16 @@
       The current system or UTC time, as returned by the system, in ISOTime2
       format.  (Introduced in 0.3.4.1-alpha.)
 
+    "rephist/ntor/onion_handshakes_requested"
+    "rephist/ntor/onion_handshakes_assigned"
+      The NTor circuit onion handshake rephist values which are requested or
+      assigned.  (Introduced in 0.4.4.1-alpha)
+
+    "rephist/tap/onion_handshakes_requested"
+    "rephist/tap/onion_handshakes_assigned"
+      The TAP circuit onion handshake rephist values which are requested or
+      assigned.  (Introduced in 0.4.4.1-alpha)
```

Comment:
So, nitpick here. `rephist` is really an internal name of `tor.git` (and not a very good one if you ask me) but regardless, I think we should expose something more meaningful to the users such as maybe: 

`stats/circuit/ntor/...`
`stats/circuit/tap/...`

Will ask more people in the net team.